tests/modules/programs/pubs/pubs-example-settings.nix

```
@@ -0,0 +1,36 @@
+{ config, lib, pkgs, ... }:
+
+{
+  config = {
+    programs.pubs = {
+      enable = true;
+
+      extraConfig = ''
+        [main]
+        pubsdir = ~/.pubs
+        docsdir = ~/.pubs/doc
+        doc_add = link
+        open_cmd = xdg-open
+
+        [plugins]
+        active = git,alias
+
+        [[alias]]
+
+        [[[la]]]
+        command = list -a
+        description = lists papers in lexicographic order
+
+        [[git]]
+        quiet = True
+        manual = False
+        force_color = False
+      '';
+    };
+
```

```suggestion
  test.stubs.pubs = { };
```

The package needs to be stubbed otherwise it will be fetched when running tests.